docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -519,522 +519,613 @@ Function: `non_zero_count`.
 
 These {anomaly-jobs} appear by default in the Anomaly Detection interface of the 
 {siem-guide}/machine-learning.html[SIEM app] in {kib}. They help you 
-automatically detect file system and network anomalies on your hosts. The list 
-below contains the jobs organized by `agent.type` (Auditbeat, Packetbeat, and 
-Winlogbeat).
+automatically detect file system and network anomalies on your hosts.
 
 // tag::siem-jobs[]
-[float]
-[[ootb-ml-jobs-siem-audit]]
-==== SIEM - Auditbeat
-
 linux_anomalous_network_activity_ecs::
++
+windows_anomalous_network_activity_ecs::
+
+Identifies Windows/Linux processes that do not usually use the network but have
+unexpected network activity, which can indicate command-and-control, lateral
+movement, persistence, or data exfiltration activity.
++
+A process with unusual network activity can denote process exploitation or
+injection, where the process is used to run persistence mechanisms that allow a
+malicious actor remote access or control of the host, data exfiltration, and
+execution of unauthorized network applications.
 
-* For network activity logs where `agent.type` is `auditbeat`.
+Job details:::
+
+* Analyzes network activity logs where `agent.type` is `auditbeat` (Linux) or `winlogbeat` (Windows).
 * Models the occurrences of processes that cause network activity.
 * Detects network activity caused by processes that occur rarely compared to 
   other processes (using the {ml-docs}/ml-rare-functions.html#ml-rare[`rare` function]).
+* {ml-docs}/ml-buckets.html[Bucket span]: 15 minutes.
```

Comment:
In my opinion, these repeated links to info about bucket spans and influencers are distracting.  I'd suggest having them appear once at the top of the page.

Reply:
I agree with this. Maybe before the first job have a short description of what each of the terms mean  (single sentence summary) with the existing hyperlink for more detailed info. And maybe have a single list of the jobs at the top of the page with links to the job itself. I'm thinking like the [Logstash mutate filter](https://www.elastic.co/guide/en/logstash/current/plugins-filters-mutate.html#plugins-filters-mutate-options)

Reply:
I think a lot of people will be using ctrl/cmd f on this page, searching for a specific job.
I don't mind though.

Reply:
@benskelker Per our chat, I think the bucket span and influencer details can be omitted from this list (and therefore the links to bucket span and influencer definitions too). One thing we might consider adding to all these pages, however, is a link to where the modules are defined: https://github.com/elastic/kibana/tree/master/x-pack/plugins/ml/server/models/data_recognizer/modules

Then folks can dig into the exact job and datafeed definitions if they so desire.  For the SIEM auditbeat ones can be found here: https://github.com/elastic/kibana/tree/master/x-pack/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat/ml

Reply:
I've merged that change in https://github.com/elastic/stack-docs/pull/933. If you want help with the merge conflict, please just let me know!